index.js

```diff
@@ -12,10 +12,27 @@ const makeDir = require('make-dir');
 const pify = require('pify');
 const pEvent = require('p-event');
 const fileType = require('file-type');
+const extName = require('ext-name');
 
 const fsP = pify(fs);
 const filenameFromPath = res => path.basename(url.parse(res.requestUrl).pathname);
 
+const getExtFromMime = res => {
+	const header = res.headers['content-type'];
+
+	if (!header) {
+		return null;
+	}
+
+	const exts = extName.mime(header);
+
+	if (exts.length !== 1) {
```

Comment:
Yes, that's intended. If it returns more than one, we can't reliably set the extension, IMO.